File: Contest 1/H_Data_Type_Guessing.cpp
#include<bits/stdc++.h>
using namespace std;
#define endl '\n'
int main() {
    long long a, b, c; cin >> a >> b >> c;
    if(ceil((a * b) /  c) != floor(a * b) /  c) cout << "double";
    else if(((a * b) /  c) > 2147483647) cout << "long long";
    else if(((a * b) /  c) < -2147483647) cout << "long long";
    else cout << "int";
    return 0;
}

File: Python/Contest 1/H_Data_Type_Guessing.py
import math

n, k, a = map(int, input().split())

if math.ceil((n * k) / a) != math.floor((n * k) / a):
    print("double")
elif ((n * k) / a) >= -2147483647 and ((n * k) / a) <= 2147483647:
    print("int")
else:
    print("long long")
